src/popup/style.css

```
@@ -1,18 +1,27 @@
 body {
+    background: #eee;
     font-family: Arial;
 }
 
 .container {
-    min-width: 500px;
+    width: 700px;
 }
 
 .hidden {
     display: none !important;
 }
 
 .mrList {
+    background: white;
+    /* border-color matches TabNav lib */
+    border-right: 1px solid rgb(225, 228, 232);
+    border-bottom: 1px solid rgb(225, 228, 232);
+    border-left: 1px solid rgb(225, 228, 232);
     overflow: auto;
-    overflow: overlay; /* This is a fix for Chrome */
+    /* This is a fix for Chrome */
+    overflow: overlay;
+    /* Override default styles */
+    padding-top: 6px !important;
```

```suggestion
    padding: 6px !important;
```
I would put this padding all around so when you hover an item it doesn't touch the sides

---

Good point, changed